app/assets/javascripts/application.js

```diff
@@ -18,3 +18,22 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+$(document).ready(function() {
+  $(".order-check").bind('change', function(){
+    if (this.checked){
+      $.ajax({
+        url: '/orders/'+this.value+'/toggle',
```

Comment:
You can use the [js-routes](https://github.com/railsware/js-routes) gem for having nice routes in javascript.

if you decide not to use it, add spaces before and after the plus sings 👍